docs/superpowers/plans/20260831_quiet_hours.md

```
@@ -0,0 +1,936 @@
+
+Expected: `In quiet hours, skipping` が出力され、通知は表示されない
+
+アプリは debug ビルドで `Timber.DebugTree()` を plant しており、タグは呼び出し元のクラス名 (`ArticleNotificationWorker`) になる。タグで絞って何も出ない場合は `adb -s <SERIAL> shell logcat -d | grep -i "quiet hours"` で確認する
+
+`<JOB_ID>` は次のコマンドで `article_notification` の周期ワークに対応するものを特定する:
+
+```bash
+adb -s <SERIAL> shell dumpsys jobscheduler | grep -A 5 "blue.starry.tokidokiroppou.staging"
+```
+
+4. 抑止時間帯を現在時刻を含まない範囲に戻し、同じ手順で通知が表示されることを確認する (抑止が過剰でないことの確認)
```

markdownlint-cli2 (0.23.2)

[warning] Ordered list item prefix
Expected: 1; Actual: 4; Style: 1/1/1

(MD029, ol-prefix)